public/app/features/explore/Logs/LogsVolumePanel.tsx

```diff
@@ -80,6 +80,7 @@ export function LogsVolumePanel(props: Props) {
         anchorToZero
         yAxisMaximum={allLogsVolumeMaximum}
         eventBus={props.eventBus}
+        showAllTimeSeries
```

Comment:
Should we also make this a state variable that the user can toggle at some point?

Reply:
whole context in https://raintank-corp.slack.com/archives/C02TXB92G6T/p1695815888660829

the TLDR is that this is just to unblock som other things, the logs volume panel can only have like 8 series i think (max 1 per each log level) so it's not really affected, we'll refactor this to make it better